### ISSUE TITLE
[icerec] Incremental Release IC2015-L3_Muon_V3 (Trac #1846)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1838]
Since there are a lot of changes in the trunk that also affects physics, the fastest way to make Muon L3 ready for production is to make another incremental release for the L3 Muon release.

Basically, we just need the IC2015-L3_Muon_V2 release but the level3-filter-muon needs to be replaced by branch [http://code.icecube.wisc.edu/projects/icecube/browser/IceCube/projects/level3-filter-muon/branches/V2015 V2015].

Migrated from https://code.icecube.wisc.edu/ticket/1846
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:18", 
    "description": "Since there are a lot of changes in the trunk that also affects physics, the fastest way to make Muon L3 ready for production is to make another incremental release for the L3 Muon release.\n\nBasically, we just need the IC2015-L3_Muon_V2 release but the level3-filter-muon needs to be replaced by branch [http://code.icecube.wisc.edu/projects/icecube/browser/IceCube/projects/level3-filter-muon/branches/V2015 V2015].", 
    "reporter": "joertlin", 
    "cc": "aturcati, mhuber", 
    "resolution": "fixed", 
    "_ts": "1550067198646291", 
    "component": "combo reconstruction", 
    "summary": "[icerec] Incremental Release IC2015-L3_Muon_V3", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2016-09-01T14:14:25", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160909T18:15:03: 
* olivas changed resolution from "" to "fixed"
* olivas changed status from "new" to "closed"
* olivas commented:

> http://code.icecube.wisc.edu/svn/meta-projects/icerec/releases/IC2015-L3_Muon_V3

**Comment by icecube-trac** [comment]
Trac update at 20160915T17:18:04: 
* joertlin changed resolution from "fixed" to ""
* joertlin commented:

> I forgot to include some other fixes because they are already a few month old (they were included into icerec releases though):
> * #1705
> * #1724
> 
> Sorry


* joertlin changed status from "closed" to "reopened"

**Comment by icecube-trac** [comment]
Trac update at 20161004T07:51:33: 
* olivas commented:

> It looks like the fix from #1724 isn't needed.  In fact, it looks like something was added which was backed out later.  It doesn't look like this made it into a release though.
> 
> http://code.icecube.wisc.edu/svn/meta-projects/icerec/releases/IC2015-L3_Muon_V4


* olivas changed status from "reopened" to "closed"
* olivas changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:18: 
* olivas changed milestone from "September 2016" to ""
* olivas commented:

> Milestone September 2016 deleted
